build/dependencies.props

```diff
@@ -43,6 +44,10 @@
     <ExternalDependency Include="Microsoft.NETCore.DotNetHostPolicy" Version="$(CoreSetupPackageVersion)" Source="$(DotNetCoreFeed)" Mirror="true" />
     <ExternalDependency Include="Microsoft.NETCore.DotNetHostResolver" Version="$(CoreSetupPackageVersion)" Source="$(DotNetCoreFeed)" Mirror="true" />
     <ExternalDependency Include="Microsoft.NETCore.Platforms" Version="2.1.0-preview1-25831-01" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Memory" Version="$(CoreFxVersion)" Source="$(DotNetCoreFeed)" Mirror="true" />
```

Comment:
Think these are alphabetically sorted. Also, there's another `System.Memory` in line 71